simulation_parameters/Constants.cs

```diff
@@ -774,6 +775,13 @@ public static class Constants
         "user://mods",
     };
 
+    /// <summary>
+    ///   Regex expresions to categorize different file types.
+    /// </summary>
+    public static readonly Regex BackupRegex = new Regex(@"^.*\.backup\.thrivesave$");
```

Comment:
These should rely on the `SAVE_EXTENSION` constant to not hard code that part.

Reply:
I wasn't quite sure how to implement this because I would have to escape "." and any other symbols that happen to be valid regex instructions

Reply:
That's why I'd use SAVE_EXTENSION as it doesn't include the dot and still shows intent that this is tightly linked to another constant.